### PR TITLE
[example] Non-convex compliant hydroelastic mesh (yellow bell pepper)

### PR DESCRIPTION
**Pull request by DamrongGuoy**
Support #17780

### CODE REVIEW

**Review by DamrongGuoy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 1 unresolved discussion, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes

*[a discussion](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCWlq09AGeOBJ78HBEr:-NCWlq0A0jFyVvCpuLtN:blhdx9a) (no related file):*
@jwnimmer-tri could you help me check what did I do wrong in `examples/hydroelastic/python_nonconvex_compliant_mesh/BUILD.bazel`, please?  Right now I have to hide this block:

```
# _VEGGIES_MESHES = forward_files(
#     srcs = ["@models_intenal//:" + x for x in veggies_mesh_files()],
#     dest_prefix = "",
#     strip_prefix = "@models_internal//:veggies/",
#     visibility = ["//visibility:private"],
# )
#
# install_data(
#     extra_prod_models = _VEGGIES_MESHES,
# )
```

Otherwise, I got this bazel error:
```
drake $ bazel run //examples/hydroelastic/python_nonconvex_compliant_mesh:drop_pepper_py 
ERROR: Traceback (most recent call last):
        File "/home/damrongguoy/GitHub/DamrongGuoy/drake/examples/hydroelastic/python_nonconvex_compliant_mesh/BUILD.bazel", line 14, column 32, in <toplevel>
                _VEGGIES_MESHES = forward_files(
        File "/home/damrongguoy/GitHub/DamrongGuoy/drake/tools/workspace/forward_files.bzl", line 28, column 17, in forward_files
                fail("'{}' not under '{}'".format(src, strip_prefix))
Error in fail: '@models_intenal//:veggies/bell_peppers/bell_pepper_no_stem_color.png' not under '@models_internal//:veggies/'
ERROR: Skipping '//examples/hydroelastic/python_nonconvex_compliant_mesh:drop_pepper_py': no such target '//examples/hydroelastic/python_nonconvex_compliant_mesh:drop_pepper_py': target 'drop_pepper_py' not declared in package 'examples/hydroelastic/python_nonconvex_compliant_mesh' defined by /home/damrongguoy/GitHub/DamrongGuoy/drake/examples/hydroelastic/python_nonconvex_compliant_mesh/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//examples/hydroelastic/python_nonconvex_compliant_mesh:drop_pepper_py': target 'drop_pepper_py' not declared in package 'examples/hydroelastic/python_nonconvex_compliant_mesh' defined by /home/damrongguoy/GitHub/DamrongGuoy/drake/examples/hydroelastic/python_nonconvex_compliant_mesh/BUILD.bazel
INFO: Elapsed time: 1.181s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

```

___

**Review by jwnimmer-tri — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 1 unresolved discussion, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @DamrongGuoy and @jwnimmer-tri)

*[a discussion](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCWlq09AGeOBJ78HBEr:-NCWqiAsEjZaQ29uCCXQ:bz14ln) (no related file):*
<details><summary><i>Previously, DamrongGuoy (Damrong Guoy) wrote…</i></summary><blockquote>

@jwnimmer-tri could you help me check what did I do wrong in `examples/hydroelastic/python_nonconvex_compliant_mesh/BUILD.bazel`, please?  Right now I have to hide this block:

```
# _VEGGIES_MESHES = forward_files(
#     srcs = ["@models_intenal//:" + x for x in veggies_mesh_files()],
#     dest_prefix = "",
#     strip_prefix = "@models_internal//:veggies/",
#     visibility = ["//visibility:private"],
# )
#
# install_data(
#     extra_prod_models = _VEGGIES_MESHES,
# )
```

Otherwise, I got this bazel error:
```
drake $ bazel run //examples/hydroelastic/python_nonconvex_compliant_mesh:drop_pepper_py 
ERROR: Traceback (most recent call last):
        File "/home/damrongguoy/GitHub/DamrongGuoy/drake/examples/hydroelastic/python_nonconvex_compliant_mesh/BUILD.bazel", line 14, column 32, in <toplevel>
                _VEGGIES_MESHES = forward_files(
        File "/home/damrongguoy/GitHub/DamrongGuoy/drake/tools/workspace/forward_files.bzl", line 28, column 17, in forward_files
                fail("'{}' not under '{}'".format(src, strip_prefix))
Error in fail: '@models_intenal//:veggies/bell_peppers/bell_pepper_no_stem_color.png' not under '@models_internal//:veggies/'
ERROR: Skipping '//examples/hydroelastic/python_nonconvex_compliant_mesh:drop_pepper_py': no such target '//examples/hydroelastic/python_nonconvex_compliant_mesh:drop_pepper_py': target 'drop_pepper_py' not declared in package 'examples/hydroelastic/python_nonconvex_compliant_mesh' defined by /home/damrongguoy/GitHub/DamrongGuoy/drake/examples/hydroelastic/python_nonconvex_compliant_mesh/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//examples/hydroelastic/python_nonconvex_compliant_mesh:drop_pepper_py': target 'drop_pepper_py' not declared in package 'examples/hydroelastic/python_nonconvex_compliant_mesh' defined by /home/damrongguoy/GitHub/DamrongGuoy/drake/examples/hydroelastic/python_nonconvex_compliant_mesh/BUILD.bazel
INFO: Elapsed time: 1.181s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

```
</blockquote></details>

See https://github.com/DamrongGuoy/drake/pull/3.

The main error was the typo `intenal` vs `internal`.  I also removed the install rule since there didn't seem to be any reason to install these files.

___

**Review by DamrongGuoy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @jwnimmer-tri)

*[a discussion](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCWlq09AGeOBJ78HBEr:-NCWsnCo5M09qqdYV9XZ:b-b17ifo) (no related file):*
<details><summary><i>Previously, jwnimmer-tri (Jeremy Nimmer) wrote…</i></summary><blockquote>

See https://github.com/DamrongGuoy/drake/pull/3.

The main error was the typo `intenal` vs `internal`.  I also removed the install rule since there didn't seem to be any reason to install these files.
</blockquote></details>

Thank you very much!  That typo charged me an hour.  I can build it now. :smile: 

I also learnt `srcs = [...] + _VEGGIES_MESHES`.

___

**Review by DamrongGuoy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 1 unresolved discussion, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @jwnimmer-tri)

___
*[`tools/workspace/models_internal/repository.bzl` line 13 at r3](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCXLdsV5geYuo2ANDgV:-NCXLdsV5geYuo2ANDgW:b-iti0az) ([raw file](https://github.com/robotlocomotion/drake/blob/98fb6049ff89312d809aeb36d1a3cfe25a5869b8/tools/workspace/models_internal/repository.bzl#L13)):*
> ```Starlark
>         local_repository_override = "/home/damrongguoy/GitHub/DamrongGuoy/RobotLocomotion_models", # noqa
>         commit = "e6bb254c786ef4d67aea7e88d141b3bb2618e964",
>         sha256 = "e246342c6d4e895dbc9d4d7206889114e6a6c99857987708f01066d839978beb",  # noqa
> ```

@jwnimmer-tri could you please tell me how to get `sha256` here?

I can get the `commit` by clicking around GitHub, but I don't know how to get the `sha256`.




_Code quote:_
```Starlark
        local_repository_override = "/home/damrongguoy/GitHub/DamrongGuoy/RobotLocomotion_models", # noqa
        commit = "e6bb254c786ef4d67aea7e88d141b3bb2618e964",
        sha256 = "e246342c6d4e895dbc9d4d7206889114e6a6c99857987708f01066d839978beb",  # noqa
```

**Review by jwnimmer-tri — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 1 unresolved discussion, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @DamrongGuoy and @jwnimmer-tri)

___
*[`tools/workspace/models_internal/repository.bzl` line 13 at r3](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCXLdsV5geYuo2ANDgV:-NCXWA-CAhNU-PJbOZfx:bhqcxls) ([raw file](https://github.com/robotlocomotion/drake/blob/98fb6049ff89312d809aeb36d1a3cfe25a5869b8/tools/workspace/models_internal/repository.bzl#L13)):*
<details><summary><i>Previously, DamrongGuoy (Damrong Guoy) wrote…</i></summary><blockquote>

@jwnimmer-tri could you please tell me how to get `sha256` here?

I can get the `commit` by clicking around GitHub, but I don't know how to get the `sha256`.


</blockquote></details>

See https://github.com/RobotLocomotion/drake/blob/master/tools/workspace/README.md#finalizing-github_archive-changes

In short, remove the sha256 line (or set it to the empty string) and bazel build will provide you the correct value.

**Review by DamrongGuoy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 1 unresolved discussion, needs platform reviewer assigned, needs at least two assigned reviewers, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits), missing label for release notes (waiting on @DamrongGuoy and @jwnimmer-tri)

___
*[`tools/workspace/models_internal/repository.bzl` line 13 at r3](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCXLdsV5geYuo2ANDgV:-NCXZDTw2jwbnsCxJovs:b-9755fn) ([raw file](https://github.com/robotlocomotion/drake/blob/98fb6049ff89312d809aeb36d1a3cfe25a5869b8/tools/workspace/models_internal/repository.bzl#L13)):*
<details><summary><i>Previously, jwnimmer-tri (Jeremy Nimmer) wrote…</i></summary><blockquote>

See https://github.com/RobotLocomotion/drake/blob/master/tools/workspace/README.md#finalizing-github_archive-changes

In short, remove the sha256 line (or set it to the empty string) and bazel build will provide you the correct value.
</blockquote></details>

Thank you!
___
*[`tools/workspace/models_internal/repository.bzl` line 12 at r4](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCXZO-AEmyPFBs1MSBP:-NCXZO-AEmyPFBs1MSBQ:b7a58i5) ([raw file](https://github.com/robotlocomotion/drake/blob/d35777709591b4f2c2d257d0fe5c03d9918ac411/tools/workspace/models_internal/repository.bzl#L12)):*
> ```Starlark
>         repository = "RobotLocomotion/models",
>         commit = "0fd6eb4b3bba6dd044ef43367cba38675483678f",
>         sha256 = "c0dcc94996da39d212cdaa7763130954551c6e9f86e73ff7751232b836c05875",  # noqa
> ```

Working temporary SHA1 until the [models PR 19](https://github.com/RobotLocomotion/models/pull/19) is merged.



_Code quote:_
```Starlark
        commit = "0fd6eb4b3bba6dd044ef43367cba38675483678f",

        sha256 = "c0dcc94996da39d212cdaa7763130954551c6e9f86e73ff7751232b836c05875",  # noqa
```

**Review by DamrongGuoy — COMMENTED** [review]
This is how it looks now. (The video is cut short to satisfy 10 MB limit).

[2022-09-25\_pepper\_bowl\_table\_short.mp4](https://files.reviewableusercontent.io/images/reviewable/42557859/zv31LbMC3jvWC4URSGXDxlVL/2022-09-25_pepper_bowl_table_short.mp4)

> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959#-:-NCrAOZe6zR-hLHYSXfL:b-gmujn1)* status: 1 unresolved discussion, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @DamrongGuoy and @jwnimmer-tri)

**Review by sherm1 — COMMENTED** [review]
Nice!

> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959#-:-NCrNXAbC-zBJ6uYmaNA:b-83sjoh)* status: 1 unresolved discussion, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @DamrongGuoy and @jwnimmer-tri)

**Review by jwnimmer-tri — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 2 unresolved discussions, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @DamrongGuoy and @jwnimmer-tri)

___
*[`examples/hydroelastic/python_nonconvex_compliant_mesh/pepper.sdf` line 43 at r5](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCuZsmc0-JkL4P6yzhf:-NCuZsmc0-JkL4P6yzhg:bkq1gtl) ([raw file](https://github.com/robotlocomotion/drake/blob/f7b6eb7f412371691f4fa896693a8240112af170/examples/hydroelastic/python_nonconvex_compliant_mesh/pepper.sdf#L43)):*
> ```
>           
>           <drake:mesh_resolution_hint>0.005</drake:mesh_resolution_hint>
> ```

I think the comment "Resolution hint is a no-op..." is a helpful piece of documentation, and we should keep it.

On the other hand, I think _actually_ adding an ignored element for the parser to chew on (and probably warn about!) does a disservice to our users.  I think putting ignored elements into demos is misleading.

**Comment by jwnimmer-tri** [comment]
I'm not sure what this is waiting on before review, but in any case +(status: do not review) to clear it from the daily report.

**Review by DamrongGuoy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 2 unresolved discussions, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @jwnimmer-tri)

*[a discussion](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NECUeEL9fjdZuF-s2mg:-NECUeEL9fjdZuF-s2mh:b-dmv383) (no related file):*
CC: @jshep-tri 

___

**Review by DamrongGuoy — COMMENTED** [review]
+@rcory for feature review, please.

@rcory This PR has the companion PR RobotLocomotion/[models/19](https://reviewable.io/reviews/robotlocomotion/models/19) that stores the VTK tetrahedral mesh of the compliant yellow bell pepper. (The SDFormat file is in this Drake PR.  The VTK file is in that `models` PR).

-(status: do not review) 

> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959#-:-NFo6MMd9I3WW4Jikapb:b-z4c7h3)* status: 2 unresolved discussions, LGTM missing from assignee rcory, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @rcory)

**Review by rcory — COMMENTED** [review]
:lgtm: 

> Reviewed 2 of 9 files at r3, 3 of 6 files at r8, 1 of 6 files at r9, 2 of 2 files at r10, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959#-:-NGJLS0M5MgJoHmk4UnL:bg7igkv)* status: 5 unresolved discussions, needs platform reviewer assigned, needs at least two assigned reviewers, missing label for release notes (waiting on @DamrongGuoy)

___
*[`examples/hydroelastic/python_nonconvex_mesh/drop_pepper.py` line 102 at r10](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NGJJFHdCxlHf3_XPHiQ:-NGJJFHeCFQvzOFRf6lu:b-x5sfpj) ([raw file](https://github.com/robotlocomotion/drake/blob/a0fd8f28f453b86ad6b44d36b53243bebb612720/examples/hydroelastic/python_nonconvex_mesh/drop_pepper.py#L102)):*
> ```Python
>         help="The fixed time step period (in seconds) of discrete updates "
>              "for the multibody plant modeled as a discrete system. "
>              "If zero, we will use an integrator for a continuous system. "
> ```

fyi, setting time_step = 0 behaves strangely at the moment. Perhaps call this out to make users aware that this example is mainly targeted at the discrete version.
___
*[`examples/hydroelastic/python_nonconvex_mesh/README.md` line 3 at r10](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NFzpScP47_3xq15AxXx:-NFzpScP47_3xq15AxXy:bokgpq) ([raw file](https://github.com/robotlocomotion/drake/blob/a0fd8f28f453b86ad6b44d36b53243bebb612720/examples/hydroelastic/python_nonconvex_mesh/README.md#L3)):*
> ```Markdown
> # Drop a bell pepper
> 
> This is a simple example of using non-convex meshes in the hydroelastic contact
> ```

nit *in* the hydroelastic -> *with* the hydroelastic
___
*[`examples/hydroelastic/python_nonconvex_mesh/README.md` line 5 at r10](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NFzpkPcBzE7yQhPigCI:-NFzpkPcBzE7yQhPigCJ:b4nd05w) ([raw file](https://github.com/robotlocomotion/drake/blob/a0fd8f28f453b86ad6b44d36b53243bebb612720/examples/hydroelastic/python_nonconvex_mesh/README.md#L5)):*
> ```Markdown
> This is a simple example of using non-convex meshes in the hydroelastic contact
> model in Python.
> The example drops a compliant-hydroelastic yellow bell pepper on a 
> ```

nit on -> onto
___
*[`examples/hydroelastic/python_nonconvex_mesh/README.md` line 9 at r10](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NGJFtzkC_InclU6r1GZ:-NGJFtzkC_InclU6r1G_:bl5ggeh) ([raw file](https://github.com/robotlocomotion/drake/blob/a0fd8f28f453b86ad6b44d36b53243bebb612720/examples/hydroelastic/python_nonconvex_mesh/README.md#L9)):*
> ```Markdown
> The bell pepper is a non-convex compliant mesh.
> The bowl is a non-convex rigid mesh.
> ```

nit The table is a compliant box primitive.

**Review by DamrongGuoy — COMMENTED** [review]
+@rpoyner-tri for platform review per Tuesday schedule, please.

> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959#-:-NGJoO9UEVW9nVbBV--G:b5j3x54)* status: 2 unresolved discussions, LGTM missing from assignee rpoyner-tri(platform), missing label for release notes (waiting on @rcory and @rpoyner-tri)

___
*[`examples/hydroelastic/python_nonconvex_mesh/drop_pepper.py` line 102 at r10](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NGJJFHdCxlHf3_XPHiQ:-NGJTh581Yqd3NhhST22:bo0eytw) ([raw file](https://github.com/robotlocomotion/drake/blob/a0fd8f28f453b86ad6b44d36b53243bebb612720/examples/hydroelastic/python_nonconvex_mesh/drop_pepper.py#L102)):*
<details><summary><i>Previously, rcory (Rick Cory) wrote…</i></summary><blockquote>

fyi, setting time_step = 0 behaves strangely at the moment. Perhaps call this out to make users aware that this example is mainly targeted at the discrete version.
</blockquote></details>

Done. Thanks.  Now I tell users to use discrete system only.
___
*[`examples/hydroelastic/python_nonconvex_mesh/README.md` line 3 at r10](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NFzpScP47_3xq15AxXx:-NGJSym42AuckNB9W8dg:boyexbg) ([raw file](https://github.com/robotlocomotion/drake/blob/a0fd8f28f453b86ad6b44d36b53243bebb612720/examples/hydroelastic/python_nonconvex_mesh/README.md#L3)):*
<details><summary><i>Previously, rcory (Rick Cory) wrote…</i></summary><blockquote>

nit *in* the hydroelastic -> *with* the hydroelastic
</blockquote></details>

Done. Thanks.
___
*[`examples/hydroelastic/python_nonconvex_mesh/README.md` line 5 at r10](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NFzpkPcBzE7yQhPigCI:-NGJT9qy0m68Fw6TecbU:boyexbg) ([raw file](https://github.com/robotlocomotion/drake/blob/a0fd8f28f453b86ad6b44d36b53243bebb612720/examples/hydroelastic/python_nonconvex_mesh/README.md#L5)):*
<details><summary><i>Previously, rcory (Rick Cory) wrote…</i></summary><blockquote>

nit on -> onto
</blockquote></details>

Done. Thanks.
___
*[`examples/hydroelastic/python_nonconvex_mesh/README.md` line 9 at r10](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NGJFtzkC_InclU6r1GZ:-NGJTB0956ciohIMaheZ:boyexbg) ([raw file](https://github.com/robotlocomotion/drake/blob/a0fd8f28f453b86ad6b44d36b53243bebb612720/examples/hydroelastic/python_nonconvex_mesh/README.md#L9)):*
<details><summary><i>Previously, rcory (Rick Cory) wrote…</i></summary><blockquote>

nit The table is a compliant box primitive.
</blockquote></details>

Done. Thanks.

**Review by rpoyner-tri — COMMENTED** [review]
:lgtm: pending test timeout, model version update.

> Reviewed 5 of 9 files at r2, 8 of 9 files at r3, 1 of 1 files at r4, 2 of 6 files at r8, 4 of 6 files at r9, 1 of 2 files at r10, 2 of 3 files at r11, 1 of 1 files at r12, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959#-:-NGOKIUF8zX2MP5z378m:be56slj)* status: 2 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits), missing label for release notes (waiting on @DamrongGuoy, @rcory, and @rpoyner-tri)

**Review by rpoyner-tri — COMMENTED** [review]
> Reviewed 1 of 1 files at r13, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 2 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits), missing label for release notes (waiting on @rcory and @rpoyner-tri)

**Review by rpoyner-tri — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 3 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits), missing label for release notes (waiting on @DamrongGuoy, @rcory, and @rpoyner-tri)

*[a discussion](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NGS7p-M8SRc753m-tQt:-NGS7p-M8SRc753m-tQu:b-ul1amz) (no related file):*
FWIW, here's the CDash test-time graph for the new test:
https://drake-cdash.csail.mit.edu/test/889651112?graph=time

So far, it's only needed a few more seconds than the old timeout of 60, and is now set to 90. Still worth considering if we can have a less slow test, but I would allow that could be in future PRs.

___

**Review by DamrongGuoy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 2 unresolved discussions, missing label for release notes (waiting on @rcory and @rpoyner-tri)

___
*[`tools/workspace/models_internal/repository.bzl` line 12 at r4](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NCXZO-AEmyPFBs1MSBP:-NGSdnoi7eUmUiTqrK5u:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/d35777709591b4f2c2d257d0fe5c03d9918ac411/tools/workspace/models_internal/repository.bzl#L12)):*
<details><summary><i>Previously, DamrongGuoy (Damrong Guoy) wrote…</i></summary><blockquote>

Working temporary SHA1 until the [models PR 19](https://github.com/RobotLocomotion/models/pull/19) is merged.
</blockquote></details>

Done.

**Review by rpoyner-tri — COMMENTED** [review]
> Reviewed 1 of 1 files at r14, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 2 unresolved discussions, missing label for release notes (waiting on @rcory)

**Review by DamrongGuoy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17959)* status: 2 unresolved discussions, missing label for release notes (waiting on @rcory and @rpoyner-tri)

*[a discussion](https://reviewable.io/reviews/robotlocomotion/drake/17959#-NGS7p-M8SRc753m-tQt:-NGSdtZZ0aFyVd7pENhd:b-88210a) (no related file):*
<details><summary><i>Previously, rpoyner-tri (Rick Poyner (rico)) wrote…</i></summary><blockquote>

FWIW, here's the CDash test-time graph for the new test:
https://drake-cdash.csail.mit.edu/test/889651112?graph=time

So far, it's only needed a few more seconds than the old timeout of 60, and is now set to 90. Still worth considering if we can have a less slow test, but I would allow that could be in future PRs.


</blockquote></details>

Thank you.  I didn't even know that we can look at the graph.  The 60+ seconds is consistent with what I saw on my Puget.

After reducing simulation_time down to 0.01 seconds, it went down to 60+ seconds as we see it now. It's doing only one time step (simulation_time = time_step = 0.01).  I suspect most of the 60+ seconds are in initialization in debug mode (a few seconds in release mode).  I'll try to reduce it in the future PR.

Thanks for the info.

___
